package-lock.json

```diff
@@ -5,7 +5,6 @@
 	"requires": true,
 	"packages": {
 		"": {
```

Comment:
What's going on in here? Should this name be deleted? Should the version be `0.1.1`?

Reply:
This file is managed by `npm`, I just ran `npm install` and it did this.

Reply:
🤯